### PR TITLE
MINOR : change whats new modal ui and added timer for github modal to appear

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- Change what's new modal ui
-  Added timer for GitHub modal to appear if what's new modal is open.

For the demo, the timer is for 5s.

https://github.com/open-metadata/OpenMetadata/assets/66266464/423fee31-2369-4cd3-8eb8-7769e0526818

Timer is for 1 min.

https://github.com/open-metadata/OpenMetadata/assets/66266464/5108933d-1af6-4484-a3cc-9ece27800beb

Timer is for 2 min.

https://github.com/open-metadata/OpenMetadata/assets/66266464/a58cbee6-c474-4146-8160-ecbd75097976

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by chirag-madlani — COMMENTED** [review]
@Ashish8689 Did you record video with different timeout?

**Comment by Ashish8689** [comment]
@chirag-madlani added 2 video for 1 and 2 min!

**Comment by chirag-madlani** [comment]
@harshach Please review this

File: openmetadata-ui/src/main/resources/ui/src/components/GithubStarCard/GithubStarCard.component.tsx [hunk 1]
```diff
@@ -71,27 +71,39 @@ const GithubStarCard = () => {
...
 
-  const handleCancel = useCallback(() => {
-    setShowGithubStarPopup(false);
-  }, []);
+  const updateGithubPopup = useCallback(
+    (show: boolean) => {
+      if (isWhatNewAlertVisible) {
+        setTimeout(() => {
+          githubPopup(show);
+        }, 120000); // timer for 2 minutes
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/GithubStarCard/GithubStarCard.component.tsx:100]: Let's use constant for this.

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/GithubStarCard/GithubStarCard.component.tsx]: Done!

**Review by harshach — APPROVED** [review]

**Pull request merged**